apps/studio/components/interfaces/Settings/API/PublicSchemaNotEnabledAlert.tsx

```diff
@@ -0,0 +1,45 @@
+import { AlertDescription_Shadcn_, AlertTitle_Shadcn_, Alert_Shadcn_, Button } from 'ui'
+import { Info } from 'lucide-react'
+import ReactMarkdown from 'react-markdown'
+import Link from 'next/link'
+import { useParams } from 'common'
+
+interface PublicSchemaNotEnabledAlertProps {
+  context: 'settings' | 'docs' // postgrest settings or autodocs page
+}
+
+const PublicSchemaNotEnabledAlert = ({ context }: PublicSchemaNotEnabledAlertProps) => {
+  const { ref: projectRef } = useParams()
+
+  return (
+    <>
+      <Alert_Shadcn_ variant="default">
+        <Info className="h-4 w-4" />
+        <AlertTitle_Shadcn_ className={context === 'docs' ? '!-mt-4' : ''}>
+          <ReactMarkdown>The `public` schema for this project is not exposed</ReactMarkdown>
+        </AlertTitle_Shadcn_>
+        <AlertDescription_Shadcn_ className={context === 'docs' ? 'grid gap-3 !-mt-7' : ''}>
```

Comment:
sorry, this nonsense required b/c docs page piles on its own custom css overriding (ruining) everything.